mediatek: Fixed phy6 DTS settings

RolandoMagico opened the pull request:
Settings for phy6 can be simplified in the DTS

aiamadeus commented:
Is your device wireless not working?
Usually only ax3000/ax4200 devices need DBDC pins.

RolandoMagico commented:
> Is your device wireless not working? Usually only ax3000/ax4200 devices need DBDC pins.

Wifi was already working before. I just compared the DTS with OEM and similar devices and saw that it is present. So if it’s not required, I can close the PR

aiamadeus commented:
> So if it’s not required, I can close the PR

The DBDC pins are not required. But I noticed something else:
```
		phy6: phy@6 {
			compatible = "maxlinear,gpy211", "ethernet-phy-ieee802.3-c45";
			reg = <6>;
			phy-mode = "2500base-x";
		};
```
Can you try this to see if it works fine?
```
		phy6: phy@6 {
			compatible = "ethernet-phy-ieee802.3-c45";
			reg = <6>;
		};
```

RolandoMagico commented:
> Can you try this to see if it works fine?

Yes, seems to work as well. Currently I'm only able to test 1G wan link, tomorrow I can check with 2.5G

aiamadeus commented:
So you can change this pr to 2.5G phy typo fix.

RolandoMagico committed:
mediatek: D-Link AQUILA PRO AI M60 A1: Fixed phy6 DTS settings

Settings for phy6 can be simplified in the DTS

Signed-off-by: Roland Reinl <reinlroland+github@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/17430
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
